wpilibNewCommands/src/main/native/include/frc2/command/TrapezoidProfileCommand.h

```diff
@@ -40,17 +40,44 @@ class TrapezoidProfileCommand
    * @param output       The consumer for the profile output.
    * @param goal The supplier for the desired state
    * @param currentState The current state
+   * @param period       The period of the command scheduler loop, in seconds.
    * @param requirements The list of requirements.
    */
   TrapezoidProfileCommand(frc::TrapezoidProfile<Distance> profile,
                           std::function<void(State)> output,
                           std::function<State()> goal,
                           std::function<State()> currentState,
+                          double period,
```

Comment:
```suggestion
                          units::second_t period,
```
The include for this is `#include <units/time.h>`.